src/status_im2/subs/onboarding.cljs

```diff
@@ -48,6 +50,22 @@
  (fn [[intro-wizard multiaccounts]]
    (recover/existing-account? (:root-key intro-wizard) multiaccounts)))
 
+(re-frame/reg-sub
+ :multiaccounts/login-profiles-picture
+ :<- [:multiaccounts/multiaccounts]
+ :<- [:mediaserver/port]
+ (fn [[multiaccounts port] [_ target-key-uid]]
+   (let [image-name (-> multiaccounts
+                        (get-in [target-key-uid :images])
+                        first
+                        :type)]
+     (when image-name
+       (image-server/get-account-image-uri {:port       port
```

Comment:
im not sure about using effect in subscription, can we dispatch`get-account-image-uri` and store result in app-db ?